question/tests/events_test.php

```diff
@@ -62,7 +62,8 @@ public function test_questions_imported() {
                 $defaultcategoryobj->id,
                 $defaultcategory,
                 null,
-                $contexts->having_cap('moodle/question:add'));
+                $contexts->having_cap('moodle/question:add'),
+                $quiz->cmid);
```

Comment:
may i get a little context of this change?

Reply:
Reverted that change.